src/rules/indentRule.ts

```diff
@@ -50,7 +50,13 @@ class IndentWalker extends Lint.RuleWalker {
         }
         var scanner = ts.createScanner(ts.ScriptTarget.ES5, false, node.text);
         var lineStarts = node.getLineStarts();
+        var endOfComment = -1;
         lineStarts.forEach((lineStart) => {
+            if (lineStart < endOfComment) {
+                // Skip checking lines inside multi-line comments
+                return;
```

Comment:
this loop has changed to a `for-of` loop in master, so this line will have to change to be `continue;`